code/campaignclassic/src/main/res/layout/push_template_manual_carousel.xml

```diff
@@ -15,44 +18,51 @@
         android:id="@+id/notification_title"
         style="@style/Title"
         android:layout_width="match_parent"
-        android:layout_height="20dp"
+        android:layout_height="wrap_content"
         android:layout_gravity="start"
         android:textAlignment="viewStart"
+        app:autoSizeTextType="uniform"
+        app:autoSizeMinTextSize="6sp"
+        app:autoSizeMaxTextSize="20sp"
+        app:autoSizeStepGranularity="2sp"
         android:layout_toLeftOf="@+id/large_icon"/>
 
     <TextView
         android:id="@+id/notification_body_expanded"
         style="@style/Body"
         android:layout_width="match_parent"
-        android:layout_height="30dp"
+        android:layout_height="wrap_content"
         android:layout_gravity="start"
         android:textAlignment="viewStart"
+        app:autoSizeTextType="uniform"
+        app:autoSizeMinTextSize="6sp"
+        app:autoSizeMaxTextSize="20sp"
+        app:autoSizeStepGranularity="2sp"
         android:layout_below="@id/notification_title"
         android:layout_toLeftOf="@+id/large_icon"/>
 
     <ViewFlipper
         android:id="@+id/manual_carousel_view_flipper"
         android:layout_width="match_parent"
         android:layout_height="300dp"
-        android:layout_gravity="center"
-        android:scaleType="centerCrop"
-        android:layout_centerInParent="true"
-        android:adjustViewBounds="true"
-        android:layout_below="@id/notification_body_expanded">
+        android:scaleType="fitCenter"
+        android:layout_below="@id/notification_body_expanded"
+        android:adjustViewBounds="true">
     </ViewFlipper>
 
     <ImageButton
         android:id="@+id/leftImageButton"
-        android:layout_width="30dp"
-        android:layout_height="40dp"
-        android:layout_marginTop="120dp"
+        android:layout_width="15dp"
+        android:layout_height="20dp"
+        android:layout_centerVertical="true"
+        android:layout_alignParentLeft="true"
         android:background="@drawable/skipleft" />
 
     <ImageButton
         android:id="@+id/rightImageButton"
-        android:layout_width="30dp"
-        android:layout_height="40dp"
-        android:layout_marginTop="120dp"
+        android:layout_width="15dp"
+        android:layout_height="20dp"
+        android:layout_centerVertical="true"
         android:layout_alignEnd="@id/manual_carousel_view_flipper"
         android:layout_alignParentEnd="true"
```

Comment:
Do we need both `layout_alignEnd` and `layout_alignParentEnd`?

Reply:
`android:layout_alignParentEnd="true"` is not needed and removed from manual and filmstrip carousels